code/WorkInProgress/pomf/spacepods/spacepods.dm

```diff
@@ -216,15 +216,15 @@
 		if("pod_civ")
 			desc = "A sleek civilian space pod."
 		if("pod_black")
-			desc = "An all black space pod with no insignias."
+			desc = "A space pod painted black."
```

Comment:
I don't know how having no insignias is worth inclusion. I'll make the text more descriptive, though.